src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -3,76 +3,73 @@ title: Proof-of-work (PoW)
 ### Proof-of-work economics {#economics}
 
-Proof-of-work is also responsible for issuing new currency into the system and incentivizing miners to do the work.
+Proof-of-work was also responsible for issuing new currency into the system and incentivizing miners to do the work.
 
-Miners who successfully create a block get rewarded with two freshly minted ETH but no longer receive all the transaction fees, as the base fee gets burned, while the tip and block reward goes to the miner. A miner may also get 1.75 ETH for an uncle block. Uncle blocks are valid blocks created by a miner practically at the same time as another miner mined the successful block. Uncle blocks usually happen due to network latency.
+Miners who successfully create a block were rewarded with two freshly minted ETH and part of the transaction fees. 1.75 ETH was also paid for an uncle block. Uncle blocks are valid blocks created by a miner practically at the same time as another miner mined the successful block. Uncle blocks usually happen due to network latency.
 
 ## Finality {#finality}
 
 A transaction has "finality" on Ethereum when it's part of a block that can't change.
 
-Because miners work in a decentralized way, two valid blocks can get mined at the same time. This creates a temporary fork. Eventually, one of these chains will become the accepted chain after a subsequent block has been mined and added, making it longer.
+Because miners worked in a decentralized way, two valid blocks could be mined at the same time. This creates a temporary fork. Eventually, one of these chains became the accepted chain after subsequent blocks were mined and added to it, making it longer.
 
-But to complicate things further, transactions rejected on the temporary fork may have been included in the accepted chain. This means it could get reversed. So finality refers to the time you should wait before considering a transaction irreversible. For Ethereum, the recommended time is six blocks or just over 1 minute. After six blocks, you can say with relative confidence that the transaction was successful. You can wait longer for even greater assurances.
-
-Finality is something to bear in mind when designing dapps. It would be a poor user experience to misrepresent transaction information to your users, especially if the transaction is of high value.
-
-Remember, this timing doesn't include the wait times for having a transaction picked up by a miner.
+But to complicate things further, transactions rejected on the temporary fork may have been included in the accepted chain. This means it could get reversed. So finality refers to the time you should wait before considering a transaction irreversible. For Ethereum, the recommended time was six blocks or just over 1 minute. After six blocks, there was high confidence that the transaction was successful - lonmgher waits gave higher confidence. Now, with proof-of-stake, finalization is an explicit, rather than probabilistic, property of a block.
```

Comment:
I think you're trying to suggest that the transaction may _not_ be included in the accepted chain:
```suggestion
To complicate things further, transactions rejected on the temporary fork may not have been included in the accepted chain. This means it could get reversed. So finality refers to the time you should wait before considering a transaction irreversible. Under the previous proof-of-work Ethereum, the recommended time was six blocks or just over 1 minute. After six blocks, there was high confidence that the transaction was successful—longer waits gave higher confidence. Now, with proof-of-stake, finalization is an explicit, rather than probabilistic, property of a block.
```

Reply:
> the recommended time was six blocks or just over 1 minute

Also, where did you get this recommendation from? Many exchanges, for example, wait closer to 30 blocks to consider a tx finalized. 

In either case, I wonder if it's worth expanding the last sentence and mentioning that under PoS "finality" will not necessary be any faster in a practical sense, but it does allow an official "finalized" state, unlike PoW which just gets less and less probable with each successive block.

Reply:
it was in the original page - it's not my addition.

Reply:
```suggestion
To complicate things further, transactions rejected on the temporary fork may not have been included in the accepted chain. This means it could get reversed. So finality refers to the time you should wait before considering a transaction irreversible. Under the previous proof-of-work Ethereum, the more blocks were mined on top of a specific block `N`, the higher confidence that the transactions in `N` were successful and would not be reverted. Now, with proof-of-stake, finalization is an explicit, rather than probabilistic, property of a block.
```